[2024-04-15T17:50:41.842+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reddit_pipeline.s3_upload manual__2024-04-15T17:50:39.020972+00:00 [queued]>
[2024-04-15T17:50:41.847+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reddit_pipeline.s3_upload manual__2024-04-15T17:50:39.020972+00:00 [queued]>
[2024-04-15T17:50:41.847+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-15T17:50:41.854+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): s3_upload> on 2024-04-15 17:50:39.020972+00:00
[2024-04-15T17:50:41.857+0000] {standard_task_runner.py:60} INFO - Started process 79 to run task
[2024-04-15T17:50:41.858+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_reddit_pipeline', 's3_upload', 'manual__2024-04-15T17:50:39.020972+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmpg3rlpy3x']
[2024-04-15T17:50:41.860+0000] {standard_task_runner.py:88} INFO - Job 44: Subtask s3_upload
[2024-04-15T17:50:41.883+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_reddit_pipeline.s3_upload manual__2024-04-15T17:50:39.020972+00:00 [running]> on host 7dff3e628787
[2024-04-15T17:50:41.917+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Kyungmin Park' AIRFLOW_CTX_DAG_ID='etl_reddit_pipeline' AIRFLOW_CTX_TASK_ID='s3_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-04-15T17:50:39.020972+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-15T17:50:39.020972+00:00'
[2024-04-15T17:50:42.122+0000] {logging_mixin.py:188} INFO - Bucket already exists
[2024-04-15T17:50:42.311+0000] {logging_mixin.py:188} INFO - File uploaded to s3
[2024-04-15T17:50:42.314+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-15T17:50:42.330+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_reddit_pipeline, task_id=s3_upload, execution_date=20240415T175039, start_date=20240415T175041, end_date=20240415T175042
[2024-04-15T17:50:42.369+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-15T17:50:42.386+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
